src/librustc_trans/Cargo.toml

```diff
@@ -27,6 +27,7 @@ rustc_incremental = { path = "../librustc_incremental" }
 rustc_llvm = { path = "../librustc_llvm" }
 rustc_platform_intrinsics = { path = "../librustc_platform_intrinsics" }
 rustc_trans_utils = { path = "../librustc_trans_utils" }
```

Comment:
What's left of `rustc_trans_utils`?

Reply:
https://github.com/MaikKlein/rust/tree/collector/src/librustc_trans_utils I am not sure where I should move the rest.

Reply:
Oh I think you can throw them into `librustc/middle` if you want to move them. But nothing I see there should retain its current form anyway.